pkg/cluster/cluster.go

```diff
@@ -138,6 +138,16 @@ func (c *Cluster) prepareBackupAndRestore() error {
 		}
 	}
 
+	// We use backup RS to mark that we have done backup creation and even restore process before.
```

Comment:
Why? If the backup deployment exists, why do we need to check backup or restore at all?